### PR TITLE
feat: update repo zip command

### PR DESCRIPTION
**Pull request by lucasdittrichzup**
### Description
This PR adds the `update repo-zip` command.

### How to verify it
Run `rit update repo-zip`

### Changelog
Add the `update repo-zip` command.

---------------------------------------------

### This pull request generated the following artifacts.
To test the health and quality of this implementation, download the respective binary for your operating system, unzip and directly run the binary like the examples below.
- **Windows**
 Download the file: **[rit-windows.zip](https://github.com/ZupIT/ritchie-cli/suites/3864130301/artifacts/96061821)**
 Unzip to some folder like: `C:\home\user\downloads\pr1046`
 Access the folder: `cd C:\home\user\downloads\pr1046`
 Directly call the binary: `.\rit.exe --version` or `.\rit.exe name of formula`
 

 - **Linux**
 Download the file: **[rit-linux.zip](https://github.com/ZupIT/ritchie-cli/suites/3864130301/artifacts/96061819)**
 Unzip to some folder like: `/home/user/downloads/pr1046`
 Access the folder: `cd /home/user/downloads/pr1046`
 Assign execute permission to binary: `chmod +x ./rit`
 Directly call the binary: `./rit --version` or `./rit name of formula`
 

 - **MacOS**
 Download the file: **[rit-macos.zip](https://github.com/ZupIT/ritchie-cli/suites/3864130301/artifacts/96061820)**
 Unzip to some folder like: `/home/user/downloads/pr1046`
 Access the folder: `cd /home/user/downloads/pr1046`
 Assign execute permission to binary: `chmod +x ./rit`
 Directly call the binary: `./rit --version` or `./rit name of formula`

> Generated at Fri Sep 24 2021 13:38:13 GMT+0000 (Coordinated Universal Time)

### CODE REVIEW

File: pkg/cmd/update_repo_zip.go [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+)
+
+/*
+	Regras:
+		- Todos os campos são de texto
+		- Quando informo ALL no update repo, qual deveria ser o comportamento?
+		- Validar:
+			- Se o repo existe no repositories.json (nome)
+			- Se a versão é diferente da informada no arquivo repositories.json
+*/
```
Comment (GuillaumeFalourd) [inline pkg/cmd/update_repo_zip.go:37]: I think you should remove this comment, or traduce it to english 👍🏼

**Review by GuillaumeFalourd — CHANGES_REQUESTED** [review]
The implementation LGTM 👍🏼 
Just update the comment you left in the update_repo_zip class.

File: pkg/cmd/update_repo_zip.go (hunk 1 above)
Comment (lucasdittrichzup) [reply pkg/cmd/update_repo_zip.go]: done!

**Review by GuillaumeFalourd — APPROVED** [review]
Well done 👍🏼

**Pull request merged**